### DISCUSSION TITLE
Density question: why `rho` becomes `b`

### DISCUSSION THREAD

**Discussion by kerim371** [slimgroup/JUDI.jl#155] [General]
Hi,

Can't understand the meaning of physical parameter `b`.
Especially if we look at the [source code](https://github.com/slimgroup/JUDI.jl/blob/f1ea0232feaa0c157ce9272f9c4760bca0f06d0d/src/TimeModeling/Types/ModelStructure.jl#L309-L313) then we can see that `b = 1f0 ./ rho` in case when `minimum(rho)/maximum(rho) > 0.1`.
In my case `minimum(rho) = 1.0f6 (g/m^3)` and `maximum(rho) = 2.5134072f6 (g/m^3)`.
As `1.0f6 / 2.5134072f6 > 0.1` thus `rho` becomes `b`.

In the comment to the commit I can see that this is done for Julia 1.8 and I'm using Julia 1.6.5

**Comment by mloubout** [comment]
`b` is the buoyancy i.e inverse of density. Makes the compute easier to reparametrize with it
